src/libs/MoneyRequestUtils.ts

```diff
@@ -71,4 +71,11 @@ function isDistanceRequest(iouType: ValueOf<typeof CONST.IOU.MONEY_REQUEST_TYPE>
     return iouType === CONST.IOU.MONEY_REQUEST_TYPE.REQUEST && selectedTab === CONST.TAB.DISTANCE;
 }
 
-export {stripCommaFromAmount, stripSpacesFromAmount, addLeadingZero, validateAmount, replaceAllDigits, isDistanceRequest};
+/**
+ * Check if scan request or not
+ */
+function isScanRequest(selectedTab: ValueOf<typeof CONST.TAB>): boolean {
```

Comment:
This is inconsistent with `isDistanceRequest` which also accepts `iouType` as param but I think that's fine as we'll be supporting split bill on scan request as well.